server/eventHandlers/profileCreation.ts

```diff
@@ -66,8 +66,15 @@ export default class extends IEventHandler {
         return;
       }
 
-      // create a new address and profile
-      addressInstance = await this._models.Address.createEmpty(chain, address);
+      // Address.createEmpty
+      const verification_token = 'NO_USER';
+      const verification_token_expires = new Date(); // expired immediately
```

Comment:
Can you explain this to me?

Reply:
We literally copied the code from Addresses to the places where the code was called. Didn't change anything. Look at `createEmpty` on master.